dav1d-sys/src/lib.rs

```diff
@@ -600,7 +611,12 @@ pub struct Dav1dSettings {
     pub strict_std_compliance: c_int,
     pub output_invisible_frames: c_int,
     pub inloop_filters: Dav1dInloopFilterType,
+    #[cfg(feature = "v1_1")]
+    pub decode_frame_type: Dav1dDecodeFrameType,
```

Comment:
We currently don't expose any setters/getters for enum-typed settings. If we want to expose these I would make a new proper Rust enum for them, which implements `TryFrom<u32>` and `Into<u32>`.

What do you think @lu-zero ?

Reply:
I'm all for it. :)